src/components/date-time/index.tsx

```diff
@@ -1,9 +1,14 @@
 import * as React from 'react'
 const { useMemo } = React
 import { format } from 'date-fns-tz'
-import { useRelativeTimeRepresent } from '../../components/hooks'
+import { useRelativeTimeRepresent } from '../hooks'
 
-export default ({ dt, className }: { dt: Date; className?: string }) => {
+export type DateTimeProps = {
+  dt: Date
+  className?: string
+}
```

Comment:
`Readonly<>`したほうがええかも(優先度低)
```suggestion
export type DateTimeProps = Readonly<{
  dt: Date
  className?: string
}>
```

Reply:
今回いじったコンポーネントの Props を Readonly にした 1e7a6e2
他のコンポーネントも別ブランチで対応しておこうかな

Reply:
たすかるかも

Reply:
https://github.com/linkage-community/mozuku/issues/757